server/channels/app/plugin_requests.go

```diff
@@ -208,6 +208,9 @@ func (ch *Channels) servePluginRequest(w http.ResponseWriter, r *http.Request, h
 		if (session != nil && session.Id != "") && err == nil && csrfCheckPassed {
 			r.Header.Set("Mattermost-User-Id", session.UserId)
 			context.SessionId = session.Id
+
+			r.Header.Del(model.HeaderAuth)
+
 		}
```

Comment:
Nit: This empty newline is not needed below `r.Header.Del`. Wondering why the linter didn't catch this, maybe we do this in some places

```suggestion
			context.SessionId = session.Id

			r.Header.Del(model.HeaderAuth)
		}
```

Reply:
I fixed the code following the suggestion!